pkg/controller/jobs/job/job_webhook.go

```diff
@@ -70,9 +79,34 @@ func (w *JobWebhook) Default(ctx context.Context, obj runtime.Object) error {
 	log.V(5).Info("Applying defaults", "job", klog.KObj(job))
 
 	jobframework.ApplyDefaultForSuspend(job, w.manageJobsWithoutQueueName)
+
+	if canDefaultManagedBy(job.Spec.ManagedBy) {
+		localQueueName, found := job.Labels[constants.QueueLabel]
+		if !found {
+			return nil
+		}
+		clusterQueueName, err := w.queues.ClusterQueueFromLocalQueue(queue.QueueKey(job.ObjectMeta.Namespace, localQueueName))
+		if err != nil {
+			log.V(5).Info("Failed to get cluster queue name", "job", klog.KObj(job), "error", err)
```

Comment:
I don't particularly like handling errors like info. IIUC "does not exist" is the only scenario in which the error is thrown. Could we instead return a bool "found" from `ClusterQueueFromLocalQueue`? Then we could ignore it similarly as we do above for LocalQueue.